examples/tina-cloud-starter/pages/_app.tsx

```diff
@@ -28,6 +28,8 @@ const App = ({ Component, pageProps }) => {
               import("react-tinacms-editor").then(({ MarkdownFieldPlugin }) => {
                 cms.plugins.add(MarkdownFieldPlugin);
               });
+
+              cms.flags.set("tina-admin", true);
```

Comment:
We should probably put this in our Docs somewhere.  (Or in the docs you make for the admin)

Reply:
Yeah, I plan to make it a special note for the `tinacms/admin` README.md. 👍